lib/lookup.js

```diff
@@ -80,11 +80,22 @@ lookup.info = function(name, region, resources, decrypt, callback) {
   function getResources(stackInfo) {
     if (!resources) return getDecrypted(stackInfo);
 
-    cfn.describeStackResources({ StackName: name }, function(err, data) {
+    var data = {};
+    cfn.listStackResources({ StackName: name }).eachPage(function(err, page) {
       if (err) return callback(new lookup.CloudFormationError('%s: %s', err.code, err.message));
 
-      stackInfo.StackResources = data.StackResources;
-      getDecrypted(stackInfo);
+      if (page) {
+        // query more pages
+        if (data && data.StackResourceSummaries) {
```

Comment:
Under what conditions would this condition not be `true`?

Reply:
I simplified this logic by initializing `data.StackResourceSummaries  = []` and always adding to that array.